src/css/loading-blocks.scss

```diff
@@ -0,0 +1,66 @@
+$base-color: #C4C4C4;
+$shine-color: #000;
+$animation-duration: 1s;
+
+@mixin background-gradient {
+  background-color: #fff;
+  background-size: 600px;
+}
+
+@mixin keyframesForLoadingBlocks($numberOfBlocks) {
+  $firstKeyframeValue: floor(floor(100%/$numberOfBlocks)/2);
+  $firstKeyframeValueForCurrentBlock: floor(floor(100/$numberOfBlocks)/2);
+  $firstKeyframeValueForCurrentBlockInPercent: #{$firstKeyframeValueForCurrentBlock} + '%';
+
+  @keyframes blocks_#{$numberOfBlocks}-shine_block1 {
+    0% {
+      background-color: #d1d1d1;
+    }
+
+    #{$firstKeyframeValue} {
+      background-color: #C4C4C4;
```

Comment:
Let's use `base-color` var or one of the colors from the `vars.scss`.

Reply:
Fixed in https://github.com/keep-network/website/pull/453/commits/3f1544bc8269ddbb364fdc0e166184a1eb99bb49